kdd-2024/ibis-tutorial-KDD2024.md

```diff
@@ -0,0 +1,240 @@
+
+- Naty Clementi
+- Gil Forsyth
+- Chloe He  
+  
+
+### Naty Clementi
+
+**Bio**
+Naty Clementi is a senior software engineer at Voltron Data, contributing to Ibis full time. She is a former academic with a Masters in Physics and PhD in Mechanical and Aerospace Engineering. She is also an active member of Pyladies and volunteers as one of the directors of Women Who Code in the Washington DC network. She is originally from Argentina, and speaks both Spanish and English fluently. 
+
+**Expertise:** 
+Naty Clementi is an experienced instructor and educator. She has presented tutorials at meetups and conferences such as SciPy, PyData NYC, PyData Seattle, PyCon US. She has also taught multiple Python courses to graduate and undergraduate engineering students at George Washington University. 
+
+The most recent Ibis tutorial is the accepted Ibis tutorial at PyCon US 2024: 
+[Tutorials: Introduction to Ibis: blazing fast analytics with DuckDB, Polars, Snowflake, and more, from the comfort of your Python repl.](https://us.pycon.org/2024/schedule/presentation/55/)
+
+
+### Gil Forsyth
+
+**Bio**
+Gil Forsyth is a staff software engineer at Voltron Data. He followed the common career path of Japanese language specialist -> administrative assistant -> mechanical engineer -> computational fluid dynamicist -> data scientist -> software engineer -> machine learning engineer -> software engineer. Gil contributes to several projects in the PyData ecosystem and is a core maintainer of xonsh and Ibis.
+
+**Expertise:** 
+Gil Forsyth is an experienced instructor, having led tutorials at several PyData conferences, PyCon, and SciPy. He also ran internal training on Python and distributed data analysis at Capital One for several years. Gil is one of the core maintainers of Ibis.
+
+The most recent Ibis tutorial is the accepted Ibis tutorial at PyCon US 2024: 
+[Tutorials: Introduction to Ibis: blazing fast analytics with DuckDB, Polars, Snowflake, and more, from the comfort of your Python repl.](https://us.pycon.org/2024/schedule/presentation/55/)
+
+
+### Chloe He
+
+**Bio**
+Chloe has a background in data science and started working on streaming systems when she was a Founding Engineer at Claypot AI, a startup tackling challenges in real-time machine learning. She led the infrastructure development of an open-source real-time feature engineering platform and worked on the translating and optimizing streaming workloads that served low-latency use cases. Later, she brought her streaming expertise to Voltron Data, where she now leads the development of Ibis streaming.
+
+**Expertise:**
+Chloe has presented talks and workshops across a number of conferences, such as DataConnect, NVIDIA GTC, and Google DevFest. She also taught lectures for the Machine Learning System Design course at Stanford and a crash course on recommendation systems.
+
+
+2. List of contributors, i.e., tutors who will only help prepare the tutorial material
+
+- Cody Peterson
```

Comment:
@lostmygithubaccount and @deepyaman  

I'm thinking of you two to help contributing with whatever we will do for ML, happy to remove you if you won't be able to help or think the other one got it.